lib/geordi/settings.rb

```diff
@@ -26,7 +26,8 @@ def initialize
 
     # Global settings
     def irb_flags
-      @global_settings['irb_flags']
+      return [@local_settings['irb_flags'].to_s, :local] if @local_settings.key? 'irb_flags'
+      [@global_settings['irb_flags'].to_s, :global] if @global_settings.key? 'irb_flags'
```

Comment:
Would be more idiomatic as:
```suggestion
      if @local_settings.key? 'irb_flags'
        [@local_settings['irb_flags'].to_s, :local] 
      elsif @global_settings.key? 'irb_flags'
        [@global_settings['irb_flags'].to_s, :global] 
      end
```